Geometry/ForwardGeometry/src/TotemGeometry.cc

```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+*
+* This is a part of TOTEM offline software.
+* Author:
+*   Laurent Forthomme
+*
+****************************************************************************/
+
+#include "Geometry/ForwardGeometry/interface/TotemGeometry.h"
+
+TotemGeometry::TotemGeometry(const DetGeomDesc* dgd) { browse(dgd, false); }
+
+void TotemGeometry::browse(const DetGeomDesc* parent, bool in_t2) {
+  if (parent->name() == "TotemT2")
+    in_t2 = true;  // define the mother volume for all children
+  if (in_t2)
+    browseT2(parent);
+  // start the recursive browsing
+  for (const auto& child : parent->components())
+    browse(child, in_t2);
+}
+
+void TotemGeometry::browseT2(const DetGeomDesc*& parent) {
+  const unsigned short arm = parent->parentZPosition() > 0. ? 0 : 1;
+  if (parent->name() == "TotemT2SupportBox")
+    addT2Plane(TotemT2DetId(arm, parent->copyno() - 1), parent);
+  else if (parent->name() == "TotemT2Scint") {
+    unsigned short plane = 2 * (parent->copyno() / 10);
+    unsigned short tile = parent->copyno() % 10;
+    if (tile % 2 != 0)
+      tile += 1;
+    else
+      plane += 1;
+    tile = tile / 2 - 1;
```

Comment:
```suggestion
    if (tile % 2 == 0)
      plane += 1;
    tile = (tile+1) / 2 - 1;
```
looks more intuitive to me, and easier to understand that once `tile = 0` is excluded the expression remains correct.

(In any case, unless there is a suggestion to restore the previous ClassVersion here above, I agree to move all these fixes in a bugfix PR that can be merged even after pre5)

Reply:
Many thanks, @perrotta ! Indeed, if we could avoid another round of `+1`s in this holiday period, things would be easier :)